ngsi_adapter/lib/adapter.js

```diff
@@ -64,7 +64,7 @@ function updateContext(reqdomain, callback) {
             updateReqOpts = {
                 hostname: remoteUrl.hostname,
                 port: remoteUrl.port,
-                path: '/NGSI10/updateContext',
+                path: '/v1/contextEntities',
```

Comment:
In order to keep backwards compatibility with older versions of Context Broker currently being used, please restrict to the standard NGSI10 API.